_chapters/02_category.md

```diff
@@ -378,39 +378,39 @@ Notice the somehow weird, (but actually completely logical) symmetry (or perhaps
 | Objects and  Morphisms        | Sets and functions |
 | **N/A**             | Element    | 
 
-By switching to external diagrams, we lose sight of the particular (the elements of our sets), but we gain the ability to see the whole universe where we were previously trapped in. In the same way as the whole realm of sets can be thought as one category, a programming language can also be thought as a category. The concept of a category allows us to find and analyze similarities between these and other structures.
+By switching to external diagrams, we lose sight of the particular elements of our sets, but we gain the ability to zoom out and see the big picture. In the same way that the whole realm of sets can be thought as one category, a programming language can also be thought of as a category. The concept of a category allows us to find and analyze similarities between these and other structures.
```

Comment:
I liked my original phrasing better, the "big picture" is just cliche, and it feels much less expressive. Do you have any specific remarks?